docs/troubleshooting.md

```diff
@@ -0,0 +1,10 @@
+## Conflicting Package Names
+
+If two Operator Sources claim the same package name the Marketplace operator does not allow the user to pick which grpc pod the operator is installed from. Instead, the Marketplace operator will randomly select one of the existing grpc pods to deploy the operator from. To avoid unexpected behavior users should avoid adding any operators with conflicting package names.
```

Comment:
nit: `operator-registry pod` instead of `grpc pod`

Also my preference is that, rather than describe the actual issue, we just mention that operator sources with conflicting package names are not supported -- I don't really see the need to describe the specific behavior of the limitation.

Reply:
Removed specific behavior in the latest commit.